src/sass/_mixins.scss

```diff
@@ -87,3 +87,14 @@
 @function rhythm($baselines) {
   @return $baselines * $baseline;
 }
+
+@mixin mintBreakpoint($name) {
+  $media: map-get($breakpointsMap, $name);
+  @if($media) {
+    @media #{$media} {
+      @content;
+    }
+  } @else {
+    @content;
```

Comment:
It should better throw some error. 
You do not want to let some typo ruin the behavior